sale_commission/models/res_partner.py

```diff
@@ -38,16 +40,18 @@ class ResPartner(models.Model):
     agent_type = fields.Selection(
         selection=[("agent", "External agent")], string="Type", required=True,
         default="agent")
-    commission = fields.Many2one(
+    commissions = fields.Many2many(
```

Comment:
Again, a migration script that moves the data present on the many2one field to the many2many one is needed.